studio/components/interfaces/BillingV2/Subscription/AddOns/AddOns.tsx

```diff
@@ -0,0 +1,277 @@
+                    onClick={() => snap.setPanelKey('computeInstance')}
+                  >
+                    Change optimized compute
+                  </Button>
+
+                  {Number(mostRecentRemainingIOBudget?.disk_io_budget) === 0 ? (
+                    <Alert
+                      withIcon
+                      className="mt-4"
+                      variant="danger"
+                      title="Your disk IO budget has run out for today"
+                    >
+                      <p>
+                        Your workload is currently running at the baseline disk IO bandwidth at{' '}
+                        {computeInstanceSpecs?.baseline_disk_io_mbs?.toLocaleString() ?? 87} Mbps
+                        and may suffer degradation in performance.
+                      </p>
+                      <p className="mt-1">
+                        Consider upgrading to a larger compute instance for a higher baseline
+                        throughput.
+                      </p>
+                    </Alert>
+                  ) : Number(mostRecentRemainingIOBudget?.disk_io_budget) <= 10 ? (
+                    <Alert
+                      withIcon
+                      className="mt-4"
+                      variant="warning"
+                      title="Your disk IO budget is running out for today"
+                    >
+                      <p>
+                        If the disk IO budget drops to zero, your workload will run at the baseline
+                        disk IO bandwidth at{' '}
+                        {computeInstanceSpecs?.baseline_disk_io_mbs?.toLocaleString() ?? 87} Mbps
+                        and may suffer degradation in performance.
+                      </p>
+                      <p className="mt-1">
+                        Consider upgrading to a larger compute instance for a higher baseline
+                        throughput.
+                      </p>
+                    </Alert>
+                  ) : null}
+
+                  <div className="mt-2 w-full flex items-center justify-between border-b py-2">
+                    <p className="text-sm text-scale-1000">Memory</p>
+                    <p className="text-sm">{computeInstanceSpecs?.memory_gb ?? 1} GB</p>
+                  </div>
+                  <div className="w-full flex items-center justify-between border-b py-2">
+                    <p className="text-sm text-scale-1000">CPU</p>
+                    <p className="text-sm">
+                      {computeInstanceSpecs?.cpu_cores ?? 2}-core ARM{' '}
+                      {computeInstanceSpecs?.cpu_dedicated ? '(Dedicated)' : '(Shared)'}
+                    </p>
+                  </div>
+                  <div className="w-full flex items-center justify-between border-b py-2">
+                    <p className="text-sm text-scale-1000">No. of connections</p>
+                    <p className="text-sm">{computeInstanceSpecs?.connections_direct ?? 60}</p>
```

Comment:
I don't think we should have any specific fallback values - rather not display anything than something wrong

Reply:
Same goes for the other specs in here